tools/src/bin/tquic_server.rs

```diff
@@ -170,6 +170,10 @@ pub struct ServerOpt {
     /// Batch size for sending packets.
     #[clap(long, default_value = "16", value_name = "NUM")]
     pub send_batch_size: usize,
+
+    /// Congestion Control Algorithm.
+    #[clap(short = 'C', long, default_value = "cubic")]
+    pub cong_control: CongestionControlAlgorithm,
```

Comment:
Why would this still exist? As we already have an option for congestion_control_algor.